include/openssl/x509err.h

```diff
@@ -101,8 +103,8 @@ int ERR_load_X509_strings(void);
 # define X509_R_BAD_X509_FILETYPE                         100
 # define X509_R_BASE64_DECODE_ERROR                       118
 # define X509_R_CANT_CHECK_DH_KEY                         114
-# define X509_R_CERT_ALREADY_IN_HASH_TABLE                101
 # define X509_R_CERTIFICATE_VERIFICATION_FAILED           139
+# define X509_R_CERT_ALREADY_IN_HASH_TABLE                101
```

Comment:
I wonder why these lines always change, does the sort order somehow depend on my
perl version or do everybody else, do something different, (i did `make update`)

Reply:
I don't know but I've seen it many times before. It's annoying.